<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAppClass</class>
 <widget class="QMainWindow" name="QtAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="COMGroupBox">
        <property name="title">
         <string>COM</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="COMComboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="COMOPushButton">
           <property name="text">
            <string>start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxBoth">
           <property name="text">
            <string>双手</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxLeft">
           <property name="text">
            <string>左手</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxRight">
           <property name="text">
            <string>右手</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="StatusGroupBox">
        <property name="title">
         <string>Status</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="TiredLabel">
           <property name="text">
            <string>疲劳度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="6">
          <widget class="QTextBrowser" name="textBrowser"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="TiredLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="BreathLabel">
           <property name="text">
            <string>呼吸</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="BreathLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="HandsStatusLabel">
           <property name="text">
            <string>心率</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="HRStatusLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="LogoLabel">
      <property name="text">
       <string>听心(上海)智能科技有限公司</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtApp.qrc"/>
 </resources>
 <connections/>
</ui>
